src/external.rs

```diff
@@ -17,7 +17,7 @@ extern "C" {
 
 impl External {
   #[inline(always)]
-  pub fn new<'s>(
+  pub unsafe fn new<'s>(
```

Comment:
Why does this need to be marked unsafe?

Reply:
> Why does this need to be marked unsafe?

```rust
error: this public function might dereference a raw pointer but is not marked `unsafe`
  --> src\external.rs:25:69
   |
25 |       scope.cast_local(|sd| v8__External__New(sd.get_isolate_ptr(), value))
   |                                                                     ^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#not_unsafe_ptr_arg_deref
   = note: `#[deny(clippy::not_unsafe_ptr_arg_deref)]` on by default
```

I'm not sure there will be any side effects with `unsafe`.

Reply:
That seems wrong, `value` is already used inside an unsafe block, seems like a false-positive from clippy

Reply:
I agree with @bartlomieju. Clippy is wrong, `value` does not get dereferenced here or anywhere in v8 for that matter.

Reply:
I try to add `#[allow(clippy::not_unsafe_ptr_arg_deref)]` code suppress this error.